packages/legacy/core/App/screens/Home.tsx

```diff
@@ -39,10 +40,11 @@ const Home: React.FC<HomeProps> = () => {
   const { start, stop } = useTour()
   const [showTourPopup, setShowTourPopup] = useState(false)
   const screenIsFocused = useIsFocused()
+  const fontScale = useFontScale()
 
   const styles = StyleSheet.create({
     flatlist: {
-      marginBottom: 35,
+      marginBottom: fontScale >= 1.7 ? 0 : 35,
```

Comment:
I think having it be 35 regardless is fine, no? Why would the font scale require the bottom margin to be removed?